script.user.js

```diff
@@ -565,7 +565,8 @@ class App {
   static updateTitle() {
     if (!ai.bar)
       return;
-    const zoom = `${Math.round(ai.scale * 100)}% - ${ai.nwidth} x ${ai.nheight} px`;
+    const megapixels = Math.round(100 * (ai.nwidth * ai.nheight / 1000000)) / 100;
```

Comment:
I usually write these long numbers as thousands `1000e3` or millions `1e6` to make the order of magnitude more obvious. What do you think?

Reply:
I totally agree. I just made the change.